etc/async_events.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="urn:magento:module:MageOS_AsyncEvents:etc/async_events.xsd">
+    <async_event name="sales.order.created">
+        <service class="Magento\Sales\Api\OrderRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="sales.order.updated">
+        <service class="Magento\Sales\Api\OrderRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="sales.order.state.updated">
+        <service class="Magento\Sales\Api\OrderRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="sales.shipment.created">
+        <service class="Magento\Sales\Api\ShipmentRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="sales.invoice.created">
+        <service class="Magento\Sales\Api\InvoiceRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="sales.creditmemo.created">
+        <service class="Magento\Sales\Api\CreditmemoRepositoryInterface" method="get"/>
+    </async_event>
+    <async_event name="customer.updated">
+        <service class="Magento\Customer\Api\CustomerRepositoryInterface" method="getById"/>
```

Comment:
I'm just jumping in. There's a known problem and a workaround with this.

Please see
https://github.com/aligent/magento2-default-async-events/pull/1#discussion_r904450099

https://github.com/aligent/magento2-default-async-events/blob/54c2943b0f08a31e56527a5c21e90f259c7d9db0/Model/AsyncCustomerManagement.php#L39

Reply:
I already noticed that, I left it for later :-) thanks for the comment! The module shouldn't be seen as a stable implementation yet, there will be more work to be done until we can mark it stable.